apps/src/netsim/NetSimRouterNode.js

```diff
@@ -751,52 +952,16 @@ NetSimRouterNode.prototype.onLogTableChange_ = function (rows) {
   }
 };
 
+/**
+ * Get list of log entries in this router's memory.
+ * @returns {NetSimLogEntry[]}
+ */
 NetSimRouterNode.prototype.getLog = function () {
   return this.myLogRowCache_.map(function (row) {
     return new NetSimLogEntry(this.shard_, row, this.packetSpec_);
   }.bind(this));
 };
 
-NetSimRouterNode.prototype.canHandleMessage_ = function (message) {
-  var fromNodeID, toNodeID, fromAddress, toAddress, simulateForNodeID,
```

Comment:
This method was killed, and sort-of resurrected below as `localSimulationOwnsMessageRow_`